content/docs/reference/templates/syntax-and-data.md

```diff
@@ -810,11 +769,11 @@ Type of variable: **{{ printf "%T" $x }}**
 
 ### templates.SDict{#templates-sdict}
 
-`templates.SDict` - This is a custom composite data type defined on an underlying data type _map\[string]interface{}._
-This is of kind _map_ having _string_ type as its key and _interface{}_ type as that key's value and can be initialized
-using `sdict` function. A map is key-value store. This means you store value and you access that value by a key. Map is
-an unordered list and the number of parameters to form key-value pairs must be even, difference to regular _map_ is that
-`templates.SDict` is ordered by its key. Retrieving specific element inside _templates.Sdict_ is by indexing its key.
+`templates.SDict` - This is a custom composite data type defined on an underlying data type _map\[string]interface{}._ This is of kind _map_ having _string_ type as its key and _interface{}_ type as that key's value and can be initialized using `sdict` function.
+A map is key-value store.
+This means you store value and you access that value by a key.
+Map is an unordered list and the number of parameters to form key-value pairs must be even, difference to regular _map_ is that `templates.SDict` is ordered by its key.
```

Comment:
```suggestion
A map is an unordered list and the number of parameters to form key-value pairs must be even, difference to regular _map_ is that `templates.SDict` is ordered by its key.
```
Is this even correct at all? I thought the custom command implementation is unordered, too, and just for debugging purposes the keys are printed in order as per Go's `String()` / `Stringer`.